### PR TITLE
Discovery

### PR DESCRIPTION
**Pull request by janegreene**
Configured basic factory bot and faker
Added link from dashboard to discover page
Added basic structure for discover page

### CODE REVIEW

File: app/views/dashboard/show.html.erb [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <h1>Welcome, <%= current_user.name %>!</h1>
 <br>
 <center>
-  <%= button_to 'Discover Movies' %>
+  <%= button_to 'Discover Movies', '/discover', method: :get %>
```
Comment (michaeljevans) [inline app/views/dashboard/show.html.erb:4]: this button is sick

File: spec/features/user/discover_spec.rb [hunk 2]
```diff
@@ -0,0 +1,17 @@
+require 'rails_helper'
+
+RSpec.describe 'After logging in' do
+  describe 'I visit dashboard and click discover button' do
+    it "takes me to discovery page" do
+      OmniAuth.config.test_mode = true
+      OmniAuth.config.mock_auth[:google_oauth2] = OmniAuth::AuthHash.new(Faker::Omniauth.google)
```
Comment (janegreene) [inline spec/features/user/discover_spec.rb:7]: Using Faker here to mock omniauth.

File: app/views/discover/index.html.erb [hunk 3]
```diff
@@ -0,0 +1,6 @@
+<center>
+  <%= button_to 'Find Top Rated Movies', '/movies', method: :get %>
+</center>
+<%= form_tag '/movies', method: :get %>
+<%= text_field_tag :keywords %>
+<%= submit_tag "Find Movies" %>
```
Comment (michaeljevans) [inline app/views/discover/index.html.erb:6]: nice basic structure for this page

File: spec/features/user/discover_spec.rb (hunk 2 above)
Comment (michaeljevans) [inline spec/features/user/discover_spec.rb:7]: this is so cool!

**Pull request merged**